.travis.yml

```diff
@@ -13,7 +13,7 @@ install:
   - npm install
 before_script:
   - mysql -e 'create database zamboni;'
-  - flake8 . --exclude='node_modules,commonplace_projects,docs,.git'
+  - flake8 . --exclude='node_modules,commonplace_projects,docs,.git,lib/es/migrations,mkt/abuse/migrations,mkt/access/migrations,mkt/api/migrations,mkt/comm/migrations,mkt/commonplace/migrations,mkt/developers/migrations,mkt/feed/migrations,mkt/files/migrations,mkt/inapp/migrations,mkt/langpacks/migrations,mkt/monolith/migrations,mkt/operators/migrations,mkt/prices/migrations,mkt/purchase/migrations,mkt/ratings/migrations,mkt/reviewers/migrations,mkt/site/migrations,mkt/submit/migrations,mkt/tags/migrations,mkt/users/migrations,mkt/versions/migrations,mkt/webapps/migrations,mkt/webpay/migrations,mkt/websites/migrations,mkt/zadmin/migrations'
```

Comment:
Yeah, we should not need to hardcode all those dirs, just `migrations` should work (and it will not exclude files named `migrations.py` so we should be fine)